### DISCUSSION TITLE
Set picture for ExtendedSplashScreen in Linux Framebuffer

### DISCUSSION THREAD

**Discussion by HakanL** [unoplatform/uno.toolkit.ui#426] [Q&A]
I'm using the ExtendedSplashScreen in my Uno application and it works great in WinUI. I can see the splash image as defined in the manifest. Now I want to make sure the same image is displayed when I'm running this on Linux FB. I see the other elements (progress ring) that I have in my Xaml, but the image seems to be set in the Toolkit somewhere and nothing is displayed when running on Linux.

How can I set the splash image for the `ExtendedSplashScreen` on Linux?

**Comment by jeromelaban** [comment]
Does the extended splashscreen work with the Gtk head for you? Is the image present in the folder structure of the published app?

**Comment by HakanL** [comment]
I don't know, I only have WinUI and FB heads. There are no splashscreen images in the bin folder. I have a barebones RPi destination without X, and limited resources so I picked the FB head. Would you recommend GTK instead?

**Reply by jeromelaban** [reply]
_Make sure to continue answering in the same thread (I know, the GitHub UX needs improvements)_

Framebuffer is fine, it's just to understand if there's something missing specifically for FB backend that the Gtk backend may not have. Can you make sure that your image is having its "Build Action" set to "Content"?

**Reply by HakanL** [reply]
_Oh, I was thinking Discussions worked like Issues, sorry about that._

Got it. Build action is set to Content (and it works in WinUI). Just as a test I copied the SplashScreen images to the Assets/Images folder on the RPi, but it didn't make a difference. I haven't done any deeper troubleshooting.

**Reply by jeromelaban** [reply]
Thanks. If you can try with Gtk head (or the Wpf head) on Windows, it will help troubleshoot further.

**Reply by HakanL** [reply]
I added Skia.Gtk and Skia.Wpf heads, and neither show the splash image, so far only WinUI shows it.

**Reply by jeromelaban** [reply]
Thanks. If you can provide a repro sample of the issue, this may help troubleshooting.

**Reply by HakanL** [reply]
Here's my WIP prototype where I'm testing out the different parts that I need for the full application. I've stripped out most irrelevant parts, hopefully this can be used to repro. https://github.com/HakanL/UnoPrototype

**Comment by nickrandolph** [comment]
This has been fixed in the latest dev builds - the docs should have been clearer that the Skia heads weren't supported in the initial release of ExtendedSplashScreen. Give it a go and report back if you're still seeing issues

**Reply by HakanL** [reply]
@nickrandolph Can you let me know what version it's implemented in, I'm currently using Uno.Toolkit.WinUI version 2.5.8 and it's not showing up. Also is there something I have to do to make it available for my Gtk/Linux head, like adding any images or links?
